scripts/tests/avh/helpers/avh_instance.py

```diff
@@ -0,0 +1,240 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+import avh_api
+import paramiko
+import websocket
+
+DEFAULT_INSTANCE_FLAVOR = "rpi4b"
+DEFAULT_INSTANCE_OS = "Ubuntu Server"
+DEFAULT_INSTANCE_OS_VERSION = "22.04.1"
+
+DEFAULT_OS_BOOTED_OUTPUT = "-----END SSH HOST KEY KEYS-----"
+
+DEFAULT_SSH_USERNAME = "pi"
+DEFAULT_SSH_PASSWORD = "raspberry"
+
+
+class AvhInstance:
+    def __init__(
+        self,
+        avh_client,
+        name,
+        flavor=DEFAULT_INSTANCE_FLAVOR,
+        os=DEFAULT_INSTANCE_OS,
+        os_version=DEFAULT_INSTANCE_OS_VERSION,
+        username=DEFAULT_SSH_USERNAME,
+        password=DEFAULT_SSH_PASSWORD,
+    ):
+        self.avh_client = avh_client
+        self.name = name
+        self.flavor = flavor
+        self.os = os
+        self.os_version = os_version
+        self.username = username
+        self.password = password
+        self.instance_id = None
+        self.console = None
+        self.ssh_pkey = None
+        self.ssh_key_id = None
+
+    def create(self):
+        self.instance_id = self.avh_client.create_instance(
+            name=self.name, flavor=self.flavor, os=self.os_version, osbuild=self.os
+        )
+
+    def wait_for_state_on(self, timeout=240):
```

Comment:
for time values, please add a unit suffix (like timeout_s or timeout_ms or similar).